### ISSUE TITLE
Bluetooth headphones block doesn't change it colors anymore

### ISSUE THREAD

**Issue by Crandel** [greshake/i3status-rust#1777]
On version 0.22 Bluetooth headphones block was changing its color based on battery level green -> yellow -> red.
Since 0.30 release it always green.
Is it possible to return old behavior? It's a critical change as I missed the time to charge them.

My config is here:

```
[[block]]
block = "bluetooth"
mac = "D4:8F:A2:8F:C6:68"
disconnected_format = ""
[block.theme_overrides]
idle_bg="#0B0B3BAA"
[block.format]
full = "$icon dh$percentage|dh"
short = "dh$percentage|dh"
```

**Comment by MaxVerevkin** [comment]
Fixed in f7f4051f2bdd13b85aa73ec044708f0e0666b238

**Comment by Crandel** [comment]
Thank you, now it works again
